src/blockchain/account.js

```diff
@@ -5,7 +5,7 @@ import Client from "../client";
 export default class Account {
   /**
    * @param {object} account
-   * @param {StellarSdk.Keypair} keypair
```

Comment:
Since mobius-client-js has no `Keypair` class, it would be better to mention Stellar SDK's methods explicitly